homeassistant/components/fronius/sensor.py

```diff
@@ -671,10 +671,9 @@ def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
         try:
             self._attr_native_value = self._get_entity_value()
-        except (KeyError, ValueError):
+        except KeyError:
```

Comment:
Where do we catch the KeyError in the other place we're calling `_get_entity_value`?

Reply:
I guess we don't catch it there as we only create entities for seen data keys.

Side note: Noticed this while browsing the integration. We should use `async_dispatcher_send` here:
https://github.com/home-assistant/core/blob/47d6d6c344b21d2afa547291c413405fbf87410e/homeassistant/components/fronius/__init__.py#L207

Reply:
Exactly.
Thank you for spotting this 👍